charts/prow/README.md

```diff
@@ -0,0 +1,34 @@
+# Prow chart 
+
+This is a helm chart for the kubernetes [test-infra prow project](https://github.com/kubernetes/test-infra/tree/master/prow).  Prow is the internal CI system/workflow engine used for public kubernetes projects.  This is an attempt to the tool for use for other github projects.  Prow was written to run in it's own kubernetes cluster.
+
+## Usage
+
+###github auth tokens
+
+One goal of this chart was to minimize the manual steps around github auth tokens and webhooks.  It uses a [tool](https://github.com/jfelten) and scripts to genreate and install a github auth token as a secret in the kubernetes cluster.  The webhooks it uses are generated by a helm a pre-install hook and removed by a helm post-install hook upon deletion.
+
+First generate a github user token and add it as a secret to the cluster:
+
+```bash
+./generate_auth_k8s.sh
+Enter GitHub username: samsungbot
+samsungbot's GitHub Password:
+wrote results to file: github_cred
+GITHUB_AUTH_ID=99013573
+GITHUB_AUTH_TOKEN=<TOKEN_VALUE>
+secret "hookmanager-cred" deleted
+secret "hookmanager-cred" created
+```
+
+Install the helm chart from the stable repo
+
+```bash
+helm repo add stable http://storage.googleapis.com/kubernetes-charts-incubator
+helm install stable/prow --set repos={"github_org/repo1" "github_org/repo2" ...}
+```
+
+## Configuration
```

Comment:
This chart was designed to minimize config and make it possible to install by specifying the repos.

We need a public facing HTTP server to serve the helm packages from.  Either a gs bucket or directly from github.  If we use github we need to use the docs dir or a branch.   The charts repo uses a gsbucket so maybe that is easier.  If want me to do a PR to set up a github site let me know. see https://github.com/kubernetes/helm/blob/master/docs/chart_repository.md.

For now I changed the names to something that will approximate what the server might be.  I am happy to discuss in the slack channel if it is easier.